examples/multisig/lib.rs

```diff
@@ -308,40 +308,54 @@ mod multisig {
         ///
         /// Since this message must be send by the wallet itself it has to be build as a
         /// `Transaction` and dispatched through `submit_transaction` and `invoke_transaction`:
-        /// ```no_run
-        /// use ink_env::{DefaultEnvironment as Env, AccountId, call::{CallParams, Selector}, test::CallData};
+        /// ```should_panic
```

Comment:
Why is the `should_panic` needed here?

Reply:
We can't actually `fire()` a call from the examples, so it'll panic (but it's fine, we just need it to compile successfully)

Reply:
Ah got it. Can you create a follow-up issue for actually running the doc tests in CI?